### ISSUE TITLE
Error during variable coercion

### ISSUE THREAD

**Issue by elad-stern** [jimkyndemeyer/js-graphql-intellij-plugin#86]
Hi everyone,

I hope I'm not a complete noob raising this flag but I must be missing something super basic here..

I'm trying to use the plugin with an int but it converts it to a float, which then fails variable coercion. Here's what I'm doing:
```
query ProductQuery($id: Int!) {
        productById(id: $id) {
            id,
    	    name
            pricing {
                price,
                pricingModel
            }
        }
    }
```
using `{ "id": 1 }` in the variables field.

I get back:
```
{
  "errors": [
    "Error during variable coercion. Violations:\n\nVariable '$id' expected value of type 'Int!' but got: 1.0. Reason: Int value expected (line 2, column 24):\n    query ProductQuery($id: Int!) {\n                       ^"
  ]
}
```

Might be worth noting that removing the variable and simply using `productById(id: 1)` works fine.

Thanks!

**Comment by jimkyndemeyer** [comment]
Hi.

Thanks for using the plugin. It looks like you're passing the variable as intended.

It might be a regression from #65. @zvorygin do you have time to investigate whether this is an issue with Gson?

On a related note, I would recommend you use the "ID" scalar to model ids in your schema. I think the cache in Relay and Apollo expects you to use that type.

**Comment by jimkyndemeyer** [comment]
This was caused by Gson always opting for Double to represent numbers.

Fixed in 1.5.4, pending JetBrains approval.
